unit/analyses/graph.cpp

```diff
@@ -0,0 +1,223 @@
+/*******************************************************************\
+
+Module: Unit test for graph class functions
+
+Author:
+
+\*******************************************************************/
+
+#include <iostream>
```

Comment:
Genuinely needed?

Reply:
Needed on line 115:
~~~
stream_message_handlert msg(std::cout);
~~~

Reply:
I'd rather have tests as quiet as possible. Will there actually be any output? If it's just output in case of errors/warnings, use a `std::ostringstream` instead of `std::cout` and check that it is empty afterwards.

Reply:
It's only used by `goto_convert(goto_model, msg)`. 

This is a direct copy+paste from the call_graph.cpp unit test and is pretty consistent with the rest of the code in the unit test dir, `std::out` seems to be pretty extensively used in the unit test directory?

Reply:
Fair enough. I have created #2372 to have this cleaned up eventually.